test/test-applications/integrations/Samples.Owin.WebApi2/Program.cs

```diff
@@ -0,0 +1,31 @@
+using System;
+using System.Net.Http;
+using System.Threading;
+using Microsoft.Owin.Hosting;
+
+namespace Samples.Owin.WebApi2
+{
+    public class Program
+    {
+        private static string Host()
+        {
+            return Environment.GetEnvironmentVariable("ASPNETCORE_URLS") ?? "http://localhost:9000/";
```

Comment:
`ASPNETCORE`?

Reply:
Yeah, that's for simplicity because the test uses the `AspNetCoreMvcTestBase` class. It doesn't really matter what variable we use, and this makes it easy!